importer/models.py

```diff
@@ -21,3 +25,12 @@ def delete_dynamic_model(instance, sender, **kwargs):
         # Removing Field Schema
     except Exception as e:
         logger.error(f"Error during deletion of Dynamic Model schema: {e.args[0]}")
+
+
+class ResourceHandler(models.Model):
+
+    """
+    Here we save the relation between the geonode resource created and the handler that created that resource
+    """
```

Comment:
This class shall include all the info for managing animported resource. The handler info is just one of these, the first to be implemented. Maybe the name should be something more generic.